<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgBackoutDialog</class>
 <widget class="QDialog" name="HgBackoutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mercurial Revision</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="revisionGroup">
     <property name="title">
      <string>Revision</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="numberButton">
        <property name="toolTip">
         <string>Select to specify a revision by number</string>
        </property>
        <property name="text">
         <string>Number</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="numberSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a revision number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight</set>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>158</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="idButton">
        <property name="toolTip">
         <string>Select to specify a revision by changeset id</string>
        </property>
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="idEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a changeset id</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="tagButton">
        <property name="toolTip">
         <string>Select to specify a revision by a tag</string>
        </property>
        <property name="text">
         <string>Tag:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="tagCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter a tag name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="branchButton">
        <property name="toolTip">
         <string>Select to specify a revision by a branch</string>
        </property>
        <property name="text">
         <string>Branch:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="branchCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter a branch name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="bookmarkButton">
        <property name="toolTip">
         <string>Select to specify a revision by a bookmark</string>
        </property>
        <property name="text">
         <string>Bookmark:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="bookmarkCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter a bookmark name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QRadioButton" name="noneButton">
        <property name="toolTip">
         <string>Select to not specify a specific revision</string>
        </property>
        <property name="text">
         <string>Parent</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Commit data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Commit message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="messageEdit">
        <property name="toolTip">
         <string>Enter the commit message or leave empty to use the default one</string>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Commit Date:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateTimeEdit" name="dateEdit">
          <property name="toolTip">
           <string>Enter optional date for the commit</string>
          </property>
          <property name="displayFormat">
           <string notr="true">yyyy-MM-dd HH:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Commit User:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="userEdit">
          <property name="toolTip">
           <string>Enter optional user for the commit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mergeCheckBox">
     <property name="toolTip">
      <string>Select to merge with parent of the project directory</string>
     </property>
     <property name="text">
      <string>Merge with current parent</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>numberButton</tabstop>
  <tabstop>numberSpinBox</tabstop>
  <tabstop>idButton</tabstop>
  <tabstop>idEdit</tabstop>
  <tabstop>tagButton</tabstop>
  <tabstop>tagCombo</tabstop>
  <tabstop>branchButton</tabstop>
  <tabstop>branchCombo</tabstop>
  <tabstop>bookmarkButton</tabstop>
  <tabstop>bookmarkCombo</tabstop>
  <tabstop>noneButton</tabstop>
  <tabstop>messageEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>mergeCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HgBackoutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HgBackoutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>numberSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>idEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tagButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>tagCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>branchButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>branchCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmarkButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>bookmarkCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
